Tests/LibWeb/test-web/TestWeb.h

```diff
@@ -59,6 +60,8 @@ struct Test {
 
     RefPtr<Gfx::Bitmap const> actual_screenshot {};
     RefPtr<Gfx::Bitmap const> expectation_screenshot {};
+
+    RefPtr<Core::Timer> timer {};
```

Comment:
Can we call this `timeout_timer`? That'd make the rest of the code a bit easier to understand imo.